LICENSE

```diff
@@ -1,6 +1,6 @@
 The MIT License (MIT)
 
-Copyright (c) 2015 OSIAM
+Copyright (C) 2016 tarent solutions GmbH
```

Comment:
Ah, this is the [LICENSE](https://github.com/osiam/osiam/blob/7e95f37dd9869a77e373d5fa800a7260ec4223e5/LICENSE) file. should I remove this?